metadata.yaml

```diff
@@ -65,6 +65,10 @@ requires:
         interface: smtp
         limit: 1
         optional: true
+    s3-backup-parameters:
```

Comment:
Maybe just `s3-backup` or `backup`? I believe it is not relevant to a user that the integration will exchange parameters.